bundles/org.eclipse.ui.editors/src/org/eclipse/ui/texteditor/AbstractDecoratedTextEditor.java

```diff
@@ -251,9 +255,11 @@ public void gotoMarker(IMarker marker) {
 	/**
 	 * The overview ruler of this editor.
 	 *
-	 * <p>This field should not be referenced by subclasses. It is <code>protected</code> for API
+	 * <p>
```

Comment:
There is a lot of unrelated javadoc format changes. Please undo, it is hard toi find code for a review

Reply:
Done